docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -383,6 +470,9 @@ Auth Service host</ScopedBlock>.
 
 ## Step 8/8. Set up systemd
 
+This section is only relevant if you are running the Executable version of the
```

Comment:
Since we're not limited to the space of a tab label here, I think we can be more explicit and say, "This section is only relevant if you are running the Teleport Slack plugin on a Linux host".